### PR TITLE
Add additionalInstrumentations docs

### PR DESCRIPTION
**Pull request by luismiramirez**
A new page in the instrumentation section that provides information on how to use the additionalInstrumentations config options so customers know how to add OpenTelemetry automatic instrumentations that we don't provide in the default set.

### CODE REVIEW

**Review by noccor — APPROVED** [review]

File: source/posts/nodejs/3.x/instrumentation/additional-instrumentations.mdx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+```js
+import { CustomInstrumentation } from "opentelemetry-custom-instrumentation";
+
+new Appsignal({
+  active: true,
+  name: "<YOUR APPLICATION NAME>",
+  pushApiKey: "<YOUR API KEY>",
+  additionalInstrumentations: [
+    new CustomInstrumentation({
+      // ws instrumentation config here
```
Comment (unflxw) [inline source/posts/nodejs/3.x/instrumentation/additional-instrumentations.mdx:27]: This shouldn't say "ws" (as in websocket)
```suggestion
      // instrumentation config here
```

File: source/posts/nodejs/3.x/instrumentation/additional-instrumentations.mdx [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+title: "Additional OpenTelemetry instrumentations"
+---
+
+<Notice>
+  🛟 Don't hesitate to [contact us](mailto:support@appsignal.com) if you run into
+  any issues while implementing custom insturmentations. We're here to help!
```
Comment (unflxw) [inline source/posts/nodejs/3.x/instrumentation/additional-instrumentations.mdx:7]:
```suggestion
  any issues while implementing custom instrumentations. We're here to help!
```

File: source/posts/nodejs/3.x/instrumentation/additional-instrumentations.mdx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+In addition to the already included instrumentations, you can also add custom ones as long as they comply with the structure of an OpenTelemetry automatic instrumentation.
+
+### Setup
+
+Using the [`additionalInstrumentations`](/nodejs/3.x/configuration/options.html#option-additionalinstrumentations) config option you can add additional instrumentations that will create spans and send them to AppSignal.
+
+Below is an example of how to use `additionalInstrumentations` in your application:
+
+```js
+import { CustomInstrumentation } from "opentelemetry-custom-instrumentation";
```
Comment (unflxw) [inline source/posts/nodejs/3.x/instrumentation/additional-instrumentations.mdx:19]: This is going into a `.cjs` file, so you don't get to use `import`:

```suggestion
const { CustomInstrumentation } = require("opentelemetry-custom-instrumentation");
```

**Review by unflxw — APPROVED** [review]

File: source/posts/nodejs/3.x/instrumentation/additional-instrumentations.mdx (hunk 1 above)
Comment (luismiramirez) [reply source/posts/nodejs/3.x/instrumentation/additional-instrumentations.mdx:27]: True, bad copypaste 😅

File: source/posts/nodejs/3.x/instrumentation/additional-instrumentations.mdx (hunk 2 above)
Comment (luismiramirez) [reply source/posts/nodejs/3.x/instrumentation/additional-instrumentations.mdx:7]: This is also a copy-paste, I'll fix other occurrences in another PR

**Pull request merged**